### PR TITLE
Bugfix loglevel selection

### PR DESCRIPTION
**Pull request by CromFr**

### CODE REVIEW

File: src/plugins/xp_bugfix/xp_bugfix.ini [hunk 1]
```diff
@@ -9,6 +16,14 @@
 
 ReplaceNetLayer = 1
 
+#
+# TODO
```
Comment (CromFr) [inline src/plugins/xp_bugfix/xp_bugfix.ini:20]: @SkywingvL I found this undocumented key. Is it still relevant ?

**Review by scottmunday84 — CHANGES_REQUESTED** [review]
It might be better to place a reference to the logger at the end of the NWNX struct that is passed to all DLLs built in NWNX4. It would be backwards compatible, and would guarantee the version of this shared variable is available to all these internal built plugins.

**Comment by CromFr** [comment]
This would mean sharing C++ objects with the plugin DLLs, and the whole point of the CPlugin ABI is to have a simple and basic interface that any language can implement.

Also xp_bugfix is not a CPlugin, it's a "legacy" C++ Plugin class ;)

**Review by scottmunday84 — APPROVED** [review]
In my original comment, I was speaking to the struct passed via the Detours API, but I can see that bugfix does things differently. I'm ok with adding an extern(al) defined variable, although overall it isn't always the best practice.

**Pull request merged**